lightly/transforms/add_grid_transform.py

```diff
@@ -0,0 +1,142 @@
+#
```

Comment:
I think we use a different license header for `lightly`, see e.g. [here](https://github.com/lightly-ai/lightly/blob/master/lightly/models/utils.py).

Reply:
fixed in https://github.com/lightly-ai/lightly/pull/1731/commits/c91fdb81474749bff9c499877062643f19bc40b1